src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+
+import duke.exceptions.DukeException;
+import duke.exceptions.InvalidCommandException;
+
+/**
+ * Processes the user's commands.
+ */
+
+public class Parser {
+
+    /**
+     * Takes in a line of user input and passes it to the TaskList to handle.
```

Comment:
should there be an empty line after the comments, before the @params?

Reply:
Thank you for the review and comments.